src/main/java/dataProviders/ConfigFileReader.java

```diff
@@ -70,4 +70,16 @@ public Boolean getBrowserWindowSize() {
         return true;
     }
 
+    public String getReportConfigPath(){
+        String reportConfigPath = properties.getProperty("reportConfigPath");
+        if(reportConfigPath!= null) return reportConfigPath;
```

Comment:
I understand that it works in this way but please add curly braces for all 'if - else' blocks

Reply:
Done